src/main/kotlin/org/opensearch/commons/sql/action/TransportQueryResponse.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.commons.sql.action
+
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.io.stream.Writeable
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import java.io.IOException
+
+/**
+ * Action Response for ppl and sql query.
+ */
+class TransportQueryResponse : BaseResponse {
+    val queryResponse: String
```

Comment:
Instead of return plain string, we talked about DataFrame/DataSet. Please create an issue to describe the idea and link to this PR.